R/deploy.R

```diff
@@ -60,6 +60,9 @@ deploy_minicran_package <- function(drat_repo,
   pkgname <- desc::desc_get("Package")
   pkgversion <- desc::desc_get_version()
 
+  # copy git config in case config is not set globally
+  file.copy(".git/config", paste0(drat_dir, "/.git/config"), overwrite = TRUE)
+
   withr::with_dir(
```

Comment:
Unrelated: `local_dir()` saves one level of indentation. Maybe it's worth splitting this function into smaller parts?

Reply:
Not entirely sure how to apply this here, feel free to tweak.